<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMultiCopyForm</class>
 <widget class="QMainWindow" name="TMultiCopyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="MultiCopy.qrc">
    <normaloff>:/MainIcon_32x32</normaloff>:/MainIcon_32x32</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="SrcBox">
       <property name="title">
        <string>Source(s)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget2" name="SrcList">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QToolButton" name="SrcAddFile">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Add File(s)</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/AddFile_16x16</normaloff>:/AddFile_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="SrcAddFolder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Add F&amp;older</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/AddFolder_16x16</normaloff>:/AddFolder_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="SrcRemove">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Remove</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Delete_16x16</normaloff>:/Delete_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="SrcClear">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Clear</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Clear_16x16</normaloff>:/Clear_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="SrcUp">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Up</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/ArrowUp_16x16</normaloff>:/ArrowUp_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="SrcDown">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Do&amp;wn</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/ArrowDown_16x16</normaloff>:/ArrowDown_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="SrcCustom"/>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="SrcAddCustom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Add_16x16</normaloff>:/Add_16x16</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="DestBox">
       <property name="title">
        <string>Destination(s)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget2" name="DestList">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="DestAddFolder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>A&amp;dd Folder</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/AddFolder_16x16</normaloff>:/AddFolder_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="DestRemove">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Re&amp;move</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Delete_16x16</normaloff>:/Delete_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="DestClear">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>C&amp;lear</string>
            </property>
            <property name="icon">
             <iconset resource="Resources.qrc">
              <normaloff>:/Clear_16x16</normaloff>:/Clear_16x16</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="DestCustom"/>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="DestAddCustom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Add_16x16</normaloff>:/Add_16x16</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="TaskSettings">
        <property name="text">
         <string>Task S&amp;ettings</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/TaskSettings_16x16</normaloff>:/TaskSettings_16x16</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Start">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Start</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Start_16x16</normaloff>:/Start_16x16</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadSourcesList"/>
    <addaction name="actionSaveSourcesList"/>
    <addaction name="actionLoadDestinationsList"/>
    <addaction name="actionSaveDestinationsList"/>
    <addaction name="actionLoadTask"/>
    <addaction name="actionSaveTask"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionStatistics"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionGeneralSettings"/>
    <addaction name="actionTaskSettings"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MultiCopy.qrc">
     <normaloff>:/MainIcon_16x16</normaloff>:/MainIcon_16x16</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/qt_16x16</normaloff>:/qt_16x16</iconset>
   </property>
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Exit_16x16</normaloff>:/Exit_16x16</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoadSourcesList">
   <property name="text">
    <string>Load sources list...</string>
   </property>
   <property name="toolTip">
    <string>Load sources list from file</string>
   </property>
  </action>
  <action name="actionLoadDestinationsList">
   <property name="text">
    <string>Load destinations list...</string>
   </property>
   <property name="toolTip">
    <string>Load destinations list from file</string>
   </property>
  </action>
  <action name="actionSaveSourcesList">
   <property name="text">
    <string>Save sources list...</string>
   </property>
   <property name="toolTip">
    <string>Save sources list to file</string>
   </property>
  </action>
  <action name="actionSaveDestinationsList">
   <property name="text">
    <string>Save destinations list...</string>
   </property>
   <property name="toolTip">
    <string>Save destinations list to file</string>
   </property>
  </action>
  <action name="actionLoadTask">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/LoadTask_16x16</normaloff>:/LoadTask_16x16</iconset>
   </property>
   <property name="text">
    <string>Load task...</string>
   </property>
   <property name="toolTip">
    <string>Load task from file</string>
   </property>
  </action>
  <action name="actionSaveTask">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/SaveTask_16x16</normaloff>:/SaveTask_16x16</iconset>
   </property>
   <property name="text">
    <string>Save task...</string>
   </property>
   <property name="toolTip">
    <string>Save task to file</string>
   </property>
  </action>
  <action name="actionGeneralSettings">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/GeneralSettings_16x16</normaloff>:/GeneralSettings_16x16</iconset>
   </property>
   <property name="text">
    <string>General Settings</string>
   </property>
  </action>
  <action name="actionTaskSettings">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/TaskSettings_16x16</normaloff>:/TaskSettings_16x16</iconset>
   </property>
   <property name="text">
    <string>Task Settings</string>
   </property>
   <property name="toolTip">
    <string>Task Settings</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="toolTip">
    <string>Statistics of a program runtime</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QListWidget2</class>
   <extends>QListWidget</extends>
   <header>QListWidget2.hpp</header>
   <slots>
    <signal>listChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SrcList</tabstop>
  <tabstop>SrcAddFile</tabstop>
  <tabstop>SrcAddFolder</tabstop>
  <tabstop>SrcRemove</tabstop>
  <tabstop>SrcClear</tabstop>
  <tabstop>SrcUp</tabstop>
  <tabstop>SrcDown</tabstop>
  <tabstop>SrcCustom</tabstop>
  <tabstop>SrcAddCustom</tabstop>
  <tabstop>DestList</tabstop>
  <tabstop>DestAddFolder</tabstop>
  <tabstop>DestRemove</tabstop>
  <tabstop>DestClear</tabstop>
  <tabstop>DestCustom</tabstop>
  <tabstop>DestAddCustom</tabstop>
 </tabstops>
 <resources>
  <include location="MultiCopy.qrc"/>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SrcCustom</sender>
   <signal>returnPressed()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>addCustomSrc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DestCustom</sender>
   <signal>returnPressed()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>addCustomDest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SrcList</sender>
   <signal>listChanged()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>srcChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DestList</sender>
   <signal>listChanged()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>destChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TaskSettings</sender>
   <signal>clicked()</signal>
   <receiver>actionTaskSettings</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SrcAddCustom</sender>
   <signal>clicked()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>addCustomSrc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DestAddCustom</sender>
   <signal>clicked()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>addCustomDest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SrcList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>srcChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DestList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>TMultiCopyForm</receiver>
   <slot>destChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCustomSrc()</slot>
  <slot>addCustomDest()</slot>
  <slot>srcChanged()</slot>
  <slot>destChanged()</slot>
 </slots>
</ui>
